typescript/sdk/src/providers/transactions/submitter/ethersV5/EV5TxSubmitterTypes.ts

```diff
@@ -0,0 +1,13 @@
+import { Address } from '@hyperlane-xyz/utils';
+
+import { ChainName } from '../../../../types.js';
+
+export interface EV5GnosisSafeTxSubmitterProps {
+  chain: ChainName;
+  safeAddress: Address;
+}
+
+export interface EV5ImpersonatedAccountTxSubmitterProps {
+  chain: ChainName;
+  userAddress: Address;
+}
```

Comment:
is chain needed for every submitter?
maybe we can omit that from each

Reply:
not actually – `EV5ImpersonatedAccountTxSubmitterProps` does not actually need a specific network tied to it, so will remove chain here. Gnosis does need it in order to verify the submitted transactions can be used with that submitter

will address as a part of https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3786